src/components/Layout/Header/index.tsx

```diff
@@ -228,6 +228,7 @@ const Header = () => {
                     onClick: () => {
                       dispatch(personaActions.cleanLogout());
                       dispatch(userActions.cleanLogout());
+                      window.sessionStorage.clear();
```

Comment:
I think it is good but did you check with the product owner is that behaviour is ok (i think we ll lost the querybuilder stuff)?

Reply:
query use the localstorage and not the sessionstorage, should have no impact :)